[2025-10-27T13:43:27.394+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-27T13:43:27.410+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_mysql_connection.test_mysql_connection manual__2025-10-27T13:38:22.282724+00:00 [queued]>
[2025-10-27T13:43:27.415+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_mysql_connection.test_mysql_connection manual__2025-10-27T13:38:22.282724+00:00 [queued]>
[2025-10-27T13:43:27.416+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-10-27T13:43:27.425+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): test_mysql_connection> on 2025-10-27 13:38:22.282724+00:00
[2025-10-27T13:43:27.431+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=255) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-27T13:43:27.433+0000] {standard_task_runner.py:64} INFO - Started process 257 to run task
[2025-10-27T13:43:27.433+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_mysql_connection', 'test_mysql_connection', 'manual__2025-10-27T13:38:22.282724+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/tes.py', '--cfg-path', '/tmp/tmpljw55v35']
[2025-10-27T13:43:27.435+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask test_mysql_connection
[2025-10-27T13:43:27.469+0000] {task_command.py:426} INFO - Running <TaskInstance: test_mysql_connection.test_mysql_connection manual__2025-10-27T13:38:22.282724+00:00 [running]> on host a0076f556b70
[2025-10-27T13:43:27.532+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_mysql_connection' AIRFLOW_CTX_TASK_ID='test_mysql_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-10-27T13:38:22.282724+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-27T13:38:22.282724+00:00'
[2025-10-27T13:43:27.534+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-27T13:43:27.540+0000] {base.py:84} INFO - Using connection ID '***_con' for task execution.
[2025-10-27T13:43:27.548+0000] {logging_mixin.py:188} INFO - MySQL Connection Test Successful!
[2025-10-27T13:43:27.549+0000] {logging_mixin.py:188} INFO - Result: (1,)
[2025-10-27T13:43:27.549+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-27T13:43:27.550+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-27T13:43:27.557+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_mysql_connection, task_id=test_mysql_connection, run_id=manual__2025-10-27T13:38:22.282724+00:00, execution_date=20251027T133822, start_date=20251027T134327, end_date=20251027T134327
[2025-10-27T13:43:27.573+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-27T13:43:27.582+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
